azure-app-service/massdriver.yaml

```
@@ -160,23 +175,29 @@ connections:
 ui:
   ui:order:
     - application
+    - command
     - image
     - dns
+    - "*"
   application:
```

Adding the `textarea` UI widget will turn the command param into a text box that is resizable, making it easier to type out larger commands and seeing what's going on.

```suggestion
    - "*"
  command:
    ui:widget: textarea
  application:
```

---

[RJSF](https://rjsf-team.github.io/react-jsonschema-form/#eyJmb3JtRGF0YSI6eyJhcHBsaWNhdGlvbiI6e30sImRucyI6eyJlbmFibGVfZG5zIjpmYWxzZX0sImltYWdlIjp7InRhZyI6ImxhdGVzdCJ9LCJmaXJzdE5hbWUiOiJDaHVjayIsImxhc3ROYW1lIjoiTm9ycmlzIiwiYWdlIjo3NSwiYmlvIjoiUm91bmRob3VzZSBraWNraW5nIGFzc2VzIHNpbmNlIDE5NDAiLCJwYXNzd29yZCI6Im5vbmVlZCJ9LCJzY2hlbWEiOnsiJGlkIjoiaHR0cHM6Ly9zY2hlbWFzLm1hc3Nkcml2ZXIuY2xvdWQvc2NoZW1hcy9idW5kbGVzLzxtZCAuTmFtZSBtZD4vc2NoZW1hLXBhcmFtcy5qc29uIiwiJHNjaGVtYSI6Imh0dHA6Ly9qc29uLXNjaGVtYS5vcmcvZHJhZnQtMDcvc2NoZW1hIiwiZGVzY3JpcHRpb24iOiI8bWQgLkRlc2NyaXB0aW9uIG1kPiIsImV4YW1wbGVzIjpbeyJfX25hbWUiOiJEZXZlbG9wbWVudCIsImFwcGxpY2F0aW9uIjp7Im1heGltdW1fd29ya2VyX2NvdW50IjozLCJtaW5pbXVtX3dvcmtlcl9jb3VudCI6MSwic2t1X25hbWUiOiJTMSJ9fSx7Il9fbmFtZSI6IlByb2R1Y3Rpb24iLCJhcHBsaWNhdGlvbiI6eyJtYXhpbXVtX3dvcmtlcl9jb3VudCI6MzAsIm1pbmltdW1fd29ya2VyX2NvdW50IjoyLCJza3VfbmFtZSI6IlAzdjMifX1dLCJwcm9wZXJ0aWVzIjp7ImFwcGxpY2F0aW9uIjp7ImRlc2NyaXB0aW9uIjoiQXBwbGljYXRpb24gY29uZmlndXJhdGlvbiIsInByb3BlcnRpZXMiOnsiY2lkciI6eyIkbWQuaW1tdXRhYmxlIjp0cnVlLCJkZXNjcmlwdGlvbiI6IlNwZWNpZnkgYSAvMjQgQ0lEUiByYW5nZSB3aXRoaW4geW91ciB2bmV0IHRvIGNyZWF0ZSBzdWJuZXQgZm9yIHRoZSBBcHAgU2VydmljZS4gVGhlIHN1Ym5ldCBDSURSIGNhbm5vdCBiZSBjaGFuZ2VkIGFmdGVyIGNyZWF0aW9uLiIsIm1lc3NhZ2UiOnsicGF0dGVybiI6Ik11c3QgYmUgYSAvMjQgcmFuZ2UgZnJvbSB3aXRoaW4gdGhlIFZOZXQgQ0lEUiJ9LCJwYXR0ZXJuIjoiXig/OjEwXFwuKD86WzAtOV18WzAtOV17Mn18MVswLTldWzAtOV18MlswLTRdWzAtOV18MjVbMC01XSl8MTcyXFwuKD86MVs2LTldfDJbMC05XXwzWzAtMV0pfDE5MlxcLjE2OCkoPzpcXC4oPzpbMC05XXxbMC05XXsyfXwxWzAtOV1bMC05XXwyWzAtNF1bMC05XXwyNVswLTVdKSl7Mn1cXC8yNCQiLCJ0aXRsZSI6IlN1Ym5ldCBDSURSIiwidHlwZSI6InN0cmluZyJ9LCJsb2NhdGlvbiI6eyIkbWQuaW1tdXRhYmxlIjp0cnVlLCJkZXNjcmlwdGlvbiI6IkF6dXJlIGxvY2F0aW9uIiwib25lT2YiOlt7ImNvbnN0Ijoid2VzdHVzIiwidGl0bGUiOiJXZXN0IFVTIn0seyJjb25zdCI6Indlc3R1czIiLCJ0aXRsZSI6Ildlc3QgVVMgMiJ9LHsiY29uc3QiOiJ3ZXN0dXMzIiwidGl0bGUiOiJXZXN0IFVTIDMifSx7ImNvbnN0IjoiZWFzdHVzIiwidGl0bGUiOiJFYXN0IFVTIn0seyJjb25zdCI6ImVhc3R1czIiLCJ0aXRsZSI6IkVhc3QgVVMgMiJ9LHsiY29uc3QiOiJub3J0aGNlbnRyYWx1cyIsInRpdGxlIjoiTm9ydGggQ2VudHJhbCBVUyJ9LHsiY29uc3QiOiJzb3V0aGNlbnRyYWx1cyIsInRpdGxlIjoiU291dGggQ2VudHJhbCBVUyJ9LHsiY29uc3QiOiJjZW50cmFsdXMiLCJ0aXRsZSI6IkNlbnRyYWwgVVMifV0sInRpdGxlIjoiTG9jYXRpb24iLCJ0eXBlIjoic3RyaW5nIn0sIm1heGltdW1fd29ya2VyX2NvdW50Ijp7ImRlc2NyaXB0aW9uIjoiTWF4aW11bSBudW1iZXIgb2YgaW5zdGFuY2VzIGZvciBhdXRvc2NhbGUuIiwibWF4aW11bSI6MzAsIm1pbmltdW0iOjEsInRpdGxlIjoiTWF4aW11bSBpbnN0YW5jZXMiLCJ0eXBlIjoiaW50ZWdlciJ9LCJtaW5pbXVtX3dvcmtlcl9jb3VudCI6eyJkZXNjcmlwdGlvbiI6Ik1pbmltdW0gbnVtYmVyIG9mIGluc3RhbmNlcyBmb3IgYXV0b3NjYWxlLiIsIm1heGltdW0iOjMwLCJtaW5pbXVtIjoxLCJ0aXRsZSI6Ik1pbmltdW0gaW5zdGFuY2VzIiwidHlwZSI6ImludGVnZXIifSwic2t1X25hbWUiOnsiZGVzY3JpcHRpb24iOiJBenVyZSBBcHAgU2VydmljZSBQbGFuIHNpemUiLCJvbmVPZiI6W3siY29uc3QiOiJTMSIsInRpdGxlIjoiMSB2Q1BVLCAxLjc1R0IgUkFNIn0seyJjb25zdCI6IlAxdjMiLCJ0aXRsZSI6IjIgdkNQVSA4R0IgUkFNIn0seyJjb25zdCI6IlAydjMiLCJ0aXRsZSI6IjQgdkNQVSAxNkdCIFJBTSJ9LHsiY29uc3QiOiJQM3YzIiwidGl0bGUiOiI4IHZDUFUgMzJHQiBSQU0ifV0sInRpdGxlIjoiU2l6ZSIsInR5cGUiOiJzdHJpbmcifX0sInJlcXVpcmVkIjpbImxvY2F0aW9uIiwiY2lkciIsInNrdV9uYW1lIiwibWluaW11bV93b3JrZXJfY291bnQiLCJtYXhpbXVtX3dvcmtlcl9jb3VudCJdLCJ0aXRsZSI6IkFwcGxpY2F0aW9uIiwidHlwZSI6Im9iamVjdCJ9LCJjb21tYW5kIjp7InRpdGxlIjoiQ29tbWFuZCIsInR5cGUiOiJzdHJpbmcifSwiZG5zIjp7ImRlcGVuZGVuY2llcyI6eyJlbmFibGVfZG5zIjp7Im9uZU9mIjpbeyJwcm9wZXJ0aWVzIjp7ImVuYWJsZV9kbnMiOnsiY29uc3QiOnRydWV9LCJzdWJkb21haW4iOnsiZGVzY3JpcHRpb24iOiJTdWJkb21haW4gZm9yIEROUyB6b25lLiBFeGFtcGxlczogJ3d3dycsICdhcGknLCAnYXBwJyIsIm1lc3NhZ2UiOnsicGF0dGVybiI6Ik11c3QgYmUgYSB2YWxpZCBzdWJkb21haW4uIE11c3QgY29udGFpbiBvbmx5IGxldHRlcnMsIG51bWJlcnMsIGRhc2hlcywgdW5kZXJzY29yZXMsIHBlcmlvZHMsIGFuZCBhc3Rlcmlza3MuIn0sInBhdHRlcm4iOiJeWy1hLXpBLVowLTkqLl8tXXsxLDYzfSQiLCJ0aXRsZSI6IlN1YmRvbWFpbiIsInR5cGUiOiJzdHJpbmcifSwiem9uZV9pZCI6eyJkZXNjcmlwdGlvbiI6IkF6dXJlIEROUyB6b25lIG5hbWUiLCJ0aXRsZSI6IkROUyBab25lIG5hbWUiLCJ0eXBlIjoic3RyaW5nIn19LCJyZXF1aXJlZCI6WyJ6b25lX2lkIiwic3ViZG9tYWluIl19LHsicHJvcGVydGllcyI6eyJlbmFibGVfZG5zIjp7ImNvbnN0IjpmYWxzZX19fV19fSwiZGVzY3JpcHRpb24iOiJETlMgY29uZmlndXJhdGlvbiIsInByb3BlcnRpZXMiOnsiZW5hYmxlX2RucyI6eyJkZWZhdWx0IjpmYWxzZSwiZGVzY3JpcHRpb24iOiJFbmFibGUgRE5TIGNvbmZpZ3VyYXRpb24gZm9yIHRoZSBhcHBsaWNhdGlvbi4iLCJ0aXRsZSI6IkVuYWJsZSBETlMiLCJ0eXBlIjoiYm9vbGVhbiJ9fSwicmVxdWlyZWQiOlsiZW5hYmxlX2RucyJdLCJ0aXRsZSI6IkROUyIsInR5cGUiOiJvYmplY3QifSwiaW1hZ2UiOnsicHJvcGVydGllcyI6eyJyZXBvc2l0b3J5Ijp7IiRtZC5pbW11dGFibGUiOnRydWUsImRlc2NyaXB0aW9uIjoiQ29udGFpbmVyIHJlcG9zaXRvcnkgdG8gdXNlIGZvciB0aGlzIGFwcGxpY2F0aW9uLiIsInRpdGxlIjoiUmVwb3NpdG9yeSIsInR5cGUiOiJzdHJpbmcifSwidGFnIjp7IiRzY2hlbWEiOiJodHRwOi8vanNvbi1zY2hlbWEub3JnL2RyYWZ0LTA3L3NjaGVtYSIsImRlZmF1bHQiOiJsYXRlc3QiLCJkZXNjcmlwdGlvbiI6IkRvY2tlciBpbWFnZSB0YWciLCJtZXNzYWdlIjp7InBhdHRlcm4iOiJUYWcgbWF5IGNvbnRhaW4gYXQgbW9zdCAxMjggY2hhcmFjdGVycyBjb21wb3NlZCBvZiBsb3dlcmNhc2UgYW5kIHVwcGVyY2FzZSBsZXR0ZXJzLCBkaWdpdHMsIHVuZGVyc2NvcmVzLCBwZXJpb2RzIGFuZCBkYXNoZXMuIFRhZyBtdXN0IHN0YXJ0IHdpdGggYW4gYWxwaGFudW1lcmljIGNoYXJhY3Rlci4ifSwicGF0dGVybiI6Il5bYS16QS1aMC05XVthLXpBLVowLTkuXy1dezAsMTI3fSQiLCJ0aXRsZSI6IkltYWdlIFRhZyIsInR5cGUiOiJzdHJpbmcifX0sInJlcXVpcmVkIjpbInJlcG9zaXRvcnkiLCJ0YWciXSwidGl0bGUiOiJDb250YWluZXIgSW1hZ2UiLCJ0eXBlIjoib2JqZWN0In19LCJyZXF1aXJlZCI6WyJhcHBsaWNhdGlvbiIsImltYWdlIiwiZG5zIl0sInRpdGxlIjoiPG1kIC5OYW1lIG1kPiJ9LCJ1aVNjaGVtYSI6eyJhcHBsaWNhdGlvbiI6eyJ1aTpvcmRlciI6WyJsb2NhdGlvbiIsImNpZHIiLCJza3VfbmFtZSIsIm1pbmltdW1fd29ya2VyX2NvdW50IiwibWF4aW11bV93b3JrZXJfY291bnQiLCIqIl19LCJjb21tYW5kIjp7InVpOndpZGdldCI6InRleHRhcmVhIn0sImRucyI6eyJ1aTpvcmRlciI6WyJlbmFibGVfZG5zIiwiem9uZV9pZCIsInN1YmRvbWFpbiIsIioiXSwiem9uZV9pZCI6eyJjbG91ZCI6ImF6dXJlIiwidWk6ZmllbGQiOiJkbnNab25lc0Ryb3Bkb3duIn19LCJpbWFnZSI6eyJ1aTpvcmRlciI6WyJyZXBvc2l0b3J5IiwidGFnIiwiKiJdfSwidWk6b3JkZXIiOlsiYXBwbGljYXRpb24iLCJjb21tYW5kIiwiaW1hZ2UiLCJkbnMiLCIqIl19LCJ0aGVtZSI6ImRlZmF1bHQiLCJsaXZlU2V0dGluZ3MiOnsidmFsaWRhdGUiOmZhbHNlLCJkaXNhYmxlIjpmYWxzZSwicmVhZG9ubHkiOmZhbHNlLCJvbWl0RXh0cmFEYXRhIjpmYWxzZSwibGl2ZU9taXQiOmZhbHNlfX0=)

---

We haven't done this elsewhere but maybe this is a good place to A/B test the textarea for a command. Let me ask around.